### ISSUE TITLE
Analysis plans Jan 10

### ISSUE THREAD

**Issue by brolosy** [corradin-lab/genetic-compensation#2]
Hey An,

Following up on our discussion during the holidays, and after reading the manuscript in more details, here is a summary of how I think we should proceed with the analysis besides what we discussed.

1- For each phenotype, using SKAT-O (which is a better statistical test that combines advantages of looking at both single variant and gene-level association), identify genes where missense mutations are associated with a Pval lower than 2.5*10-8 (-log10Pval of 7.602), but nonsense mutations do not meet this Pval.  We should not worry about the Beta Burden for now.

2- Once we have a list of genes with their associated phenotypes we can get the CSVs for each gene and exclude genes where nonsense mutations are associated with a meaningful phenotype (this is to avoid genes where missense and nonsense lead to different phenotypes and can thereby be caught from step 1 as nonsense being not associated with a phenotype).

3- Exclude genes which are more constrained in gnomad in LoF compared to missense.

4- Exclude genes associated with phenotypes like standing height etc.

Let me know what you think.

**Comment by hoangthienan95** [comment]
Hey I have uploaded the code and result for the analysis here.

TLDR: Found 384 genes after going through step 1-> 3. Will have to do step 4 with you manually

![image](https://user-images.githubusercontent.com/25307953/150520113-54daab33-61de-4c90-b4ff-4d948065ca90.png)

You can view commentary of each step and the results here:

Step 1: #3 
Step 2: #4 
Step3: #1 

Let me know if you have any comments. Talk soon!
